.github/workflows/sonarqube.yml

```diff
@@ -24,14 +17,7 @@ permissions:
 
 jobs:
   build:
-    # Forked repos do not have access to the Sonar account
-    if: github.repository == 'spotbugs/sonar-findbugs'
-    environment: |
-        (github.event_name != 'pull_request_target' && 'internal') || (
```

Comment:
This patch is associated to keepign the pull requets target which is now removed.  I'm not quite sure this was entirely accurate as it failed on internal builds and I think it should have worked or at least gave a better error than failure to understand the logic.